src/network-uptime/network-dashboard.py

```diff
@@ -29,7 +29,7 @@
 
 @st.cache
 def load_device_data(device):
-    api = f"{BASE_API_URL}/monitor/devices/uptime?tenant=airqo&startDate=2022-01-01T00:00:00.001Z&endDate=2022-03-25T19:00:00.001Z&device_name=' + device
+    api = f"{BASE_API_URL}/monitor/devices/uptime?tenant=airqo&startDate=2022-01-01T00:00:00.001Z&endDate=2022-03-25T19:00:00.001Z&device_name=" + device
```

Comment:
@123MwanjeMike @uman95 any reason as to why we are hard cording the dates?

Reply:
That's still in progress
@uman95 mentioned that he'll add the ability to dynamically select ranges later on.

Reply:
cool @123MwanjeMike. Could we change the PR to Draft or update the PR labels to reflect its status

Reply:
Ohh.
This PR was specifically meant to fix the application runtime error.
@uman95 is working on the dynamic date selection and error handling on another branch.

Reply:
cool @123MwanjeMike